File: IoT_Cosmos_Trigger/IoT_Cosmos_Trigger/Alert.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IoT_Cosmos_Trigger
{
    public class Alert
    {
        public string deviceID { get; set; }
        public string alertType { get; set; }
        public string lat { get; set; }
        public string lng { get; set; }
        public bool isActive { get; set; }
        public DateTime createdAt { get; set; }
    }
}

File: backend/models/Alert.js
const mongoose = require('mongoose');

const alertSchema = new mongoose.Schema({
    createdAt: String,
    deviceID: String,
    lat: String,
    lng: String,
    alertType: String,
    isActive: Boolean,
},{ timestamps: true });

module.exports = mongoose.model('Alert', alertSchema);